api/openai/v1/chat.go

```diff
@@ -0,0 +1,87 @@
+package v1
+
+import (
+	openai "github.com/sashabaranov/go-openai"
+)
+
+type ChatCompletionRequest struct {
+	openai.ChatCompletionRequest
+}
+
+func (r *ChatCompletionRequest) GetModel() string {
```

Comment:
How would we implement something like a Model router with this approach? https://github.com/lm-sys/RouteLLM

Reply:
Will follow up on this in a later PR